content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -1,15 +1,31 @@
 ---
 title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+spelling: cSpell:ignore venv distro rolldice randint 
+spelling: cSpell:ignore rollspan loglevel loggingexporter
 weight: 1
 ---
 
-In this page, you'll learn how to set up and get tracing telemetry from an HTTP
-server with Flask. If you're not using Flask, that's fine - this guide will also
-work with Django, FastAPI,
-[and more](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation).
+This page will show you how to get started with OpenTelemetry in Python.
+
+You will learn how you can instrument a simple application automatically, in
+such a way that [traces][], [metrics][] and [logs][] are emitted to the console.
+
+## Prerequisites
+
+Ensure that you have the following installed locally:
+
+- [Python 3](https://www.python.org/)
+
+## Example Application
+
+The following example uses a basic [Flaks](https://flask.palletsprojects.com/)
+application. If you're not using Flask, that's fine — this guide will also work
+with Django, FastAPI
+[and more](/ecosystem/registry/?component=instrumentation&language=python)
```

Comment:
```suggestion
application. If you're not using Flask, that's ok — the example also works
with Django, FastAPI, and 
[more](/ecosystem/registry/?component=instrumentation&language=python)
```

In the statement above, what is "more" referring to? More what? Consider adding a word or two to clarify.

Reply:
Thanks for calling this out, I think this sentence needs to be rephrased entirely, like:

> If you are not using Flask, that's ok — you can use OpenTelemetry with many other popular frameworks like Django or FastAPI as well! You can find a list of supported libraries in the [OpenTelemetry registry](/ecosystem/registry/?component=instrumentation&language=python)

WDYT?

Reply:
How about this?

> If you are not using Flask, that's ok — you can use OpenTelemetry Python with other web frameworks as well, such as Django and FastAPI. For a complete list of libraries for supported frameworks, see the [regsitry](#).

Reply:
```suggestion
application. If you are not using Flask, that's ok — you can use OpenTelemetry Python with other web frameworks as well, such as Django and FastAPI. For a complete list of libraries for supported frameworks, see the [regsitry](https://github.com/ecosystem/registry/?component=instrumentation&language=python).
```